src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/resource/mapper/ActionPlanDomainMapper.kt

```diff
@@ -0,0 +1,9 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.resource.mapper
+
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.domain.goal.ActionPlan
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.resource.model.ActionPlanResponse
+
+// TODO - placeholder until MapStruct dependencies are merged in
```

Comment:
Ah yes, I see why you want some of my stuff merging in 👍 
I think you also want my branch so that you have the `DomainConfiguration` class, which is where you'll new up the domain `ActionPlanService` into a bean